sdk/eventhub/event-hubs/test/eventProcessor.spec.ts

```diff
@@ -875,12 +880,10 @@ describe("Event Processor", function(): void {
       const events: EventData[] = [];
 
       for (const partitionId of partitionIds) {
-        const producer = client.createProducer({ partitionId });
         for (let index = 1; index <= 100; index++) {
           events.push({ body: `${expectedMessagePrefix} ${index} ${partitionId}` });
         }
-        await producer.send(events);
-        await producer.close();
+        await producerClient.sendBatch(events, { partitionId });
```

Comment:
Ah, seeing this change reminds me how far we've come from the early preview days. I'm so glad we don't ask users to instantiate a new client per partition anymore!